lib/stats.js

```diff
@@ -67,7 +67,7 @@ var stats = {
       var childrens = options.childrens ? stat.cutime + stat.cstime : 0
       var total = 0;
 
-      if(history.utime) {
+      if(typeof history.utime !== 'undefined') {
```

Comment:
If you dont change that if statement, history.utime=0 will still go in the else.  Are you proposing that the if/else statement be removed altogether and replaced with that new assignment?  I think that would work.